test/spec/general.js

```diff
@@ -211,9 +211,9 @@ define(function(require) {
 
       setupVerifyIdTokenTest({
         title: 'verifies a valid idToken using issuer option',
-        idToken: tokens.standardIdToken,
+        idToken: tokens.verifiableIdToken,
         verifyOpts: {
-          issuer: 'https://auth-js-test.okta.com'
+          issuer: 'https://lboyette.trexcloud.com'
```

Comment:
Unfortunately, this is the one spot I can't get rid of my org-specific domain. The token that's generated by jwt.io doesn't have the same signature that Okta generates, so `tokens.verifiableIdToken` is a token that was actually generated by Okta. The other tokens we use in our tests are valid jwts, but don't pass our (jwks) signature validation.

If we can figure out how to generate a valid jwks for the jwt.io tokens, all of our jwts would have valid signatures. We'd need to change the key in https://github.com/okta/okta-auth-js/blob/master/test/xhr/keys.js.

I'll open a separate ticket for this